gsitemap.php

```diff
@@ -805,7 +794,7 @@ public function createSitemap($id_shop = 0)
         if ($this->cron) {
             die();
         }
-        header('Location: ./index.php?tab=AdminModules&configure=gsitemap&token='.Tools::getAdminTokenLite('AdminModules').'&tab_module='.$this->tab.'&module_name=gsitemap&validation');
+        Tools::redirectAdmin('index.php?tab=AdminModules&configure=gsitemap&token='.Tools::getAdminTokenLite('AdminModules').'&tab_module='.$this->tab.'&module_name=gsitemap&validation');
```

Comment:
I think it's better to use `getAdminLink()` as explained before.

Reply:
can you rewrite the code snippets to use getAdminLink(), I am afraid to mess something up. I can't translate the code from the preview example.